back/app/models/user.rb

```diff
@@ -287,6 +288,14 @@ def blank_and_can_be_deleted?
     sso? && email.blank? && new_email.blank? && password_digest.blank? && identity_ids.count == 1
   end
 
+  # Used to reject auth if the security level in the token is lower than the user's security level
+  # Can add verification in here as a level 2 maybe
```

Comment:
It's a really smart idea to store some security state in JWT.

Do I understand correctly that if a user has JWT with a security level lower than the current one in the DB, they will be logged out?

> Can add verification in here as a level 2 maybe

So, if a user is logged in in two browsers, and then they verify their account in browser 1, they will be logged out from browser 2?

I'm also wondering what can be other use cases for the security level being a number? Maybe we could be more explicit and store a state like `confirmation_required: true/false`.

Reply:
> So, if a user is logged in in two browsers, and then they verify their account in browser 1, they will be logged out from browser 2?

It will have the same effect as logging them out in browser 2 - as the security level is lower then the backend will not return a user when presented with that JWT.

I went for a 'security-level' as a name it is deliberately both vague and flexible so we don't end up being very specific with attributes in the JWT

Reply:
Maybe we could still give the levels names in addition to the number, as opposed to just numbers, to make it easier to understand?